app/views/declaration/additionalActors/additional_actors_add.scala.html

```diff
@@ -37,9 +37,11 @@
         errorSummary: errorSummary,
         sectionHeader: sectionHeader,
         exportsInputText: exportsInputText,
-        tariffDetails: tariffDetails,
+        tariffExpander: tariffExpander,
         saveButtons: saveButtons,
-        formHelper: uk.gov.hmrc.play.views.html.helpers.FormWithCSRF)
+        formHelper: uk.gov.hmrc.play.views.html.helpers.FormWithCSRF,
+        appConfig: AppConfig
```

Comment:
Maybe better inject `AppConfig` as implicit? This is a practise in some other places.
You could then avoid passing it to `tariffExpander` explicitly.

Reply:
Yes but I'd have to alter all the calling controllers to have AppConfig as an implicit parameter (so basically adding more code to them to save a little in the templates). So I'm not sure I see the benefit of changing approaches tbh.

Reply:
I did consider this but I'd have to alter every controller also to add AppConfig as an implicit param and as time is short I've decided the small gain isn't worth the effort.